test/integration_test/common_test.go

```diff
@@ -70,11 +72,45 @@ func validateStorageCluster(
 	cluster *corev1alpha1.StorageCluster,
 	timeout, interval time.Duration,
 ) error {
-	// Validate expected portworx pods are running and no more, no less
-	return validateStorageClusterPods(cluster, timeout, interval)
+	var err error
+	cluster, err = k8s.Instance().GetStorageCluster(cluster.Name, cluster.Namespace)
+	if err != nil && !errors.IsNotFound(err) {
+		return err
+	}
+
+	if err = validateStorageClusterPods(cluster, timeout, interval); err != nil {
+		return err
+	}
+
+	svc, err := k8s.Instance().GetService("portworx-service", cluster.Namespace)
+	pxEndpoint := ""
+	if err == nil {
+		pxEndpoint = svc.Spec.ClusterIP
+	}
+
+	conn, err := grpc.Dial(pxEndpoint, grpc.WithInsecure())
```

Comment:
The endpoint would also require the sdk port. You can get that from the above service itself.